### PR TITLE
Improve ratings in review boxes

### PR DESCRIPTION
**Pull request by mvidalgarcia**

### CODE REVIEW

**Comment by pferreir** [comment]
I think something got broken:

image

File: indico/htdocs/sass/modules/abstracts/_abstract_page.scss [hunk 1]
```diff
@@ -384,3 +404,18 @@
         }
     }
 }
+
+.rating-questions > div.form-group {
```
Comment (ThiefMaster) [inline indico/htdocs/sass/modules/abstracts/_abstract_page.scss]: is the `div` needed to specificity reasons? if not i'd omit it

File: indico/modules/events/abstracts/templates/forms/rating_widget.html [hunk 2]
```diff
@@ -19,5 +19,16 @@
             var $widget = $this.closest('.rating-widget');
             $widget.find('.rating-value').text($this.val());
         });
+
+        $('#{{ field.id }} label').on('mouseenter', function() {
+            var $this = $(this);
+            var $widget = $this.closest('.rating-widget');
+            $widget.find('.rating-value').text($this.data('value'));
+        })
+        .on('mouseleave', function() {
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/forms/rating_widget.html]: merge into previous line

File: indico/modules/events/abstracts/templates/forms/rating_widget.html [hunk 3]
```diff
@@ -19,5 +19,16 @@
...
+            var $this = $(this);
+            var $widget = $this.closest('.rating-widget');
+            $widget.find('.rating-value').text($(':checked', $widget).val() || '-');
```
Comment (ThiefMaster) [inline indico/modules/events/abstracts/templates/forms/rating_widget.html]: `$widget.find(':checked')`

**Comment by pferreir** [comment]
(rebased)

**Pull request merged**